tutorials/en_us/Setting up your minecraft server.md

```diff
@@ -21,9 +21,18 @@ PAUSE
 
 ![server jar and run file](https://i.imgur.com/RDvuoer.png)
 
-1024M is how many megabytes are allocated to the server, 1024M or 2048M is good for our purpose. Save the file and close your text editor, go to the file in File Explorer and right-click, rename, and highlight the file extension "txt" and type "bat". This changes it to a batch file so we can run the server jar. We are ready to run the server now! Double click the run.bat file and it should come up with a new terminal window and after a while say "Agree to the Minecraft EULA". close out of the terminal window and find a file called "eula.txt" that was generated. Open it and change "eula=false" to "eula=true", save it then double click run.bat again. It may take a while for all the files to generate so be patient, once it's done your server should be good to go!
+1024M is how many megabytes are allocated to the server, 1024M or 2048M is good for our purpose. Save the file and close your text editor, go to the file in File Explorer and right-click, rename, and highlight the file extension "txt" and type "bat". This changes it to a batch file so we can run the server jar. This is not needed but it a good idea.
+
+-Xmx1024M is the maximum amount of ram that the server will be running, in this case 1GB.
+-Xms1024M is the Initial (Starting) amount of ram that the server will be running, in this case 1GB.
```

Comment:
Might be a good idea to say "1024MB" instead of "1GB" just to help those who don't know the conversions, so can't see the connection.

Reply:
Noting that 1024 ≠ 1GB, but 1024MB = 1GiB

Reply:
Actually 1024MiB = 1GiB

Reply:
yeah that sounds more correct LOL

Reply:
So any changes needed?